index.html

```diff
@@ -24,8 +24,8 @@
       </button>
     </div>
     <a class="pop-window" href="#Portfolio">Portfolio</a>
-    <a class="pop-window" href="#about">about</a>
-    <a class="pop-window" href="#footer"> conctact</a>   
+    <a class="pop-window" href="#about">About</a>
+    <a class="pop-window" href="#footer"> Conctact</a>   
```

Comment:
- [Optional ] As suggested by the previous reviewer please make sure that  you Paid more attention to detail always type correct spellings for "contact", Please adjust it.
Your design Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/195124209-2afd9177-d946-4707-86bc-5cf0a55a321d.PNG)